docs/user-guide/filtering-layers.md

```diff
@@ -99,10 +99,15 @@ In order to filter a numerical filed matching the records *greater than* or *equ
 
 #### Quick Filter by clicked point
 
-If the user wants to filter records in the Attribute Table simply by clicking on the map, he can activate the  **Filter clicking on the map** <img src="../img/button/filter_geometry_button.jpg" class="ms-docbutton"/> button (the button became blue) and then click on the map over the features he need to select. The list of records in the [Attribute Table](attributes-table.md) will be automatically filtered by matching with the selected point in the map.
+If the user wants to filter records in the Attribute Table simply by clicking on the map, he can activate the  **Filter clicking on the map** <img src="../img/button/filter_geometry_button.jpg" class="ms-docbutton"/> button (the button became blue) and then click on the map over the features he need to select and can add features from the filter list by pressing Ctrl + click. 
+The list of records in the [Attribute Table](attributes-table.md) will be automatically filtered by matching with the selected points in the map.
```

Comment:
The list of records in the [Attribute Table](attributes-table.md) will be automatically filtered according to such user selection.